docs/api-reference/README.md

```diff
@@ -0,0 +1,42 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+# API Reference
+
+Use the following reference docs to understand the kubernetes REST API for various API group versions:
+
+* v1: [operations](https://htmlpreview.github.io/?https://github.com/kubernetes/kubernetes/blob/HEAD/docs/api-reference/v1/operations.html), [model definitions](https://htmlpreview.github.io/?https://github.com/kubernetes/kubernetes/blob/HEAD/docs/api-reference/v1/definitions.html)
```

Comment:
Yes this doc should be copied to kubernetes.io whenever we push docs.
I guess that should happen in a few weeks whenever we have docs added/updated for new 1.2 features. cc @kubernetes/docs 

In versionize-docs, we replace HEAD by release version: https://github.com/kubernetes/kubernetes/blob/0e229d59f09c0f221eb45ad193f5c0f8ef875fc9/build/versionize-docs.sh#L67. We can probably update it to replace by kubernetes.io links instead. I am not sure what the kubernetes.io link is going to be. Filed https://github.com/kubernetes/kubernetes/issues/23159 to keep track.